recommend/report_text.go

```diff
@@ -59,7 +60,7 @@ func (r TextReport) SectionEnd(img *ImageInfo) error {
 func (r TextReport) Record(ms MatchSpec, policyName string) error {
 	var rec []string
 
-	rec = append(rec, policyName)
+	rec = append(rec, wrapPolicyName(policyName, 35))
```

Comment:
It is possible to set the column width with TableWriter. ([ref](https://github.com/olekukonko/tablewriter/blob/74c60be0ef6886d331ebc3c91b2ac0a7971715b6/table.go#L194))

Reply:
I fought with the settings available in `TableWriter` before writing this function. I found out `TableWriter` wraps the text to the next line only after end of a word (when encountering a `space` char). Since `policyName` is a single word, it doesn't gets wrapped and the columns gets misaligned in the CLI output. That's why I ended up writing this function.